_data/organizations.yaml

```diff
@@ -1,87 +0,0 @@
-cloudflare:
```

Comment:
does eleventy not speak yaml? i find multi-line stuff hard to read in json, so i'd vote yaml if possible, but nbd either way.

Reply:
I agree that YAML is easier to read, but [YAML data files are not supported in core](https://github.com/11ty/eleventy/pull/624#issuecomment-515519263) and I didn't want to add a plugin/dependency for that without discussion.